client/src/components/brushableHistogram/index.tsx

```diff
@@ -38,12 +38,33 @@ interface BrushableHistogramOwnProps {
   isUserDefined?: boolean;
   isGeneSetSummary?: boolean;
   field: string;
+  onGeneExpressionComplete: () => void;
+  zebra?: boolean;
+  mini?: boolean;
+  width?: number;
+  setGenes?: Map<string, boolean>;
```

Comment:
These prop types were missing previously but masked by the inclusion of any in the props type